lib/renderer/web-view/web-view-impl.ts

```diff
@@ -65,7 +63,8 @@ export class WebViewImpl {
     iframeElement.style.flex = '1 1 auto';
     iframeElement.style.width = '100%';
     iframeElement.style.border = '0';
-    v8Util.setHiddenValue(iframeElement, 'internal', this);
+    // used by RendererClientBase::IsWebViewFrame
+    (iframeElement as any).isWebView = true;
```

Comment:
As discussed we should find a way to make this property not-impersonatable, either by checking "is this frame a webview" in another way, or by using a `symbol` instead here.

Reply:
I've removed this change from this PR to avoid mixing it with a different change. I will address this separately